rhasspyclient/__main__.py

```diff
@@ -105,7 +108,7 @@ async def main():
 
     # Begin client session
     async with aiohttp.ClientSession() as session:
-        client = RhasspyClient(args.api_url, session)
+        client = RhasspyClient(args.api_host, args.api_port, session)
```

Comment:
The `secure` parameter you added to `RhasspyClient.__init__` isn't used in the main function. Shouldn't you add an argument to the argument parser to use https/wss? I also think `tls` (and the corresponding `--tls` argument) is more descriptive than `secure`.

Reply:
I'd agree with that. It'll be a little while before I can get back to this but I can fix when I have time